source/adapters/hip/enqueue.cpp

```diff
@@ -1895,3 +1895,34 @@ void setCopyRectParams(ur_rect_region_t Region, const void *SrcPtr,
                      : (DstType == hipMemoryTypeDevice ? hipMemcpyHostToDevice
                                                        : hipMemcpyHostToHost));
 }
+
+UR_APIEXPORT ur_result_t UR_APICALL urEnqueueTimestampRecordingExp(
+    ur_queue_handle_t hQueue, bool blocking, uint32_t numEventsInWaitList,
+    const ur_event_handle_t *phEventWaitList, ur_event_handle_t *phEvent) {
+
```

Comment:
Is this function missing a scopedContext?

Reply:
Good catch! It has been added.